README.md

```diff
@@ -1,46 +1,56 @@
-# 👷‍♀ ‍Compiled
+# Compiled
+
+**Build time atomic CSS in JS without the runtime cost.
```

Comment:
is CSS in JS normally hyphenated?

Reply:
i prefer it not to be tbh - whats ur thoughts

Reply:
Unsure and unfussed. CSS-in-JS sort of denotes it more as a 'thing' which it kinda is, but happy to defer to whatever you've applied across the codebase / in docs.